### PR TITLE
Laptop Casting: reusing all the laptop parts

### PR DESCRIPTION
**Pull request by CRImier**
**TL;DR**

I do all kinds of stuff with laptop parts and want to show all the old and new ways of reusing them in your projects.

**Long Version**

Apart from the obvious stuff (HDDs, RAM sticks etc.) I've personally reused keyboards (as external keyboards with a Teensy controller+custom PCB), LCD panels as additional monitors, broken LCD panels as lightboxes, as lasercutting materials, and I've reused the laptop LED strips for lighting. I've also reused a laptop touchscreen as a USB touchscreen together with a FullHD LCD panel, essentially making an external portable touch-enabled monitor. I've rebuilt working laptop motherboards into embedded PCs of sorts in various projects, and I stockpile broken motherboards as component sources for all kinds of projects (the N-FETs are quite good, but there's 1000 and 1 other components). Laptop chargers can be used for powering all kinds of projects - and even for charging 2-3 laptops at once when you're working with multiple different laptops at your desk, for example. I've used laptop fans for additional cooling in projects with i.e. power transistors - one can also use heatsink+heatpipe+fan combos for cooling i.e. MOSFETs in higher-power applications, I haven't needed that yet but I keep it in mind. I've reused a laptop WiFi+BT cards as a reliable USB-BT adapter when a cheap USB-BT dongle failed on me and I needed one Raspberry Pi project to be Bluetooth-enabled urgently. Plus, of course, there's laptop webcams - really easy to repurpose, I've made a tutorial on that. There's something I'm forgetting, for sure - I'll be going through my old notes and pictures for the presentation, as well as working on some new things (hopefully).

Oh right, you can recycle laptop batteries for 18650 and flat cells. I haven't looked at touchpads yet, but I will be this year - some are PS2 which is a bit tricky, but newer ones are I2C+INT and that's more compatible with i.e. a Raspberry Pi.

### CODE REVIEW

**Comment by Pawka** [comment]
@CRImier Can you please provide more details or some examples what you've made from parts? I just want a bit more details in the description. Thanks!

**Comment by CRImier** [comment]
Sure! Apart from the obvious stuff (HDDs, RAM sticks etc.) I've personally reused keyboards (as external keyboards [with a Teensy controller+custom PCB](https://www.tindie.com/products/crimier/eee-keyboard-to-usb-with-firmware-for-teensy20/)), LCD panels as additional monitors, broken LCD panels as lightboxes, as lasercutting materials, and I've reused the laptop LED strips for lighting. I've also reused a laptop touchscreen as a USB touchscreen together with a FullHD LCD panel, essentially making an external portable touch-enabled monitor. I've rebuilt working laptop motherboards into embedded PCs of sorts in various projects, and I stockpile broken motherboards as component sources for all kinds of projects (the N-FETs are quite good, but there's 1000 and 1 other components). Laptop chargers can be used for powering all kinds of projects - and even for charging 2-3 laptops at once when you're working with multiple different laptops at your desk, for example. I've used laptop fans for additional cooling in projects with i.e. power transistors - one can also use heatsink+heatpipe+fan combos for cooling i.e. MOSFETs in higher-power applications, I haven't needed that yet but I keep it in mind. I've reused a laptop WiFi+BT cards as a reliable USB-BT adapter when a cheap USB-BT dongle failed on me and I needed one Raspberry Pi project to be Bluetooth-enabled urgently. Plus, of course, there's laptop webcams - really easy to repurpose, [I've made a tutorial on that](https://hackaday.io/project/110436-laptop-webcam-reuse-made-simple). There's something I'm forgetting, for sure - I'll be going through my old notes and pictures for the presentation, as well as working on some new things (hopefully).

**Comment by Pawka** [comment]
Thanks! I've edited description and title of this PR so it would be easier to find information for folks. Are you going to make presentation? or presentation and some workshop/demo?

**Comment by CRImier** [comment]
Can't imagine running a successful workshop on this, I'd rather try to make light of as much stuff as possible; it will be a presentation - 30 to 45 minutes, if that's OK.

**Comment by Pawka** [comment]
Presentation is totally fine. I just wanted to understand your plans better ;-)
Please target to 30 mins. Thanks!
